packages/-ember-caluma/app/templates/demo/form-rendering.hbs

```diff
@@ -1,6 +1,9 @@
 {{#if @model}}
   <CfContent
     @documentId={{@model}}
-    @context={{hash actionButtonOnSuccess=this.actionButtonOnSuccess}}
+    @context={{hash
+      actionButtonOnSuccess=this.actionButtonOnSuccess
+      showValidation=false
```

Comment:
`false` is the default, you can remove this line.

Reply:
Removed it.